src/Storage/StorageClient.php

```diff
@@ -4,13 +4,13 @@
 
 class StorageClient extends StorageBucket
 {
-	public function __construct($api_key, $reference_id)
+	public function __construct($api_key, $reference_id, $domain, $scheme, $path)
```

Comment:
After correcting the `__contstructor` we can remove all these extra params - it should work by default then.
```
public function __construct($api_key, $reference_id, $domain = 'supabase.co', $scheme = 'https', $path = '/storage/v1')
```